analysis/config/testdata/full-config.yaml

```diff
@@ -37,15 +37,14 @@ options:
   # packages that do not match the regex will not be analyzed
   pkg-filter: "analysis.*"
 
-  # For context-sensitive graph reachability based analysis this parameter can be used to limit the callstack depth
-  # that can be considered during the analysis. This can be used for debugging, or to artifically only filter
-  # short traces
-  max-depth: 42
-
   # this can be set to limit the number of alarms raised by an analysis if you only care about the truth value of
   # the output, or only want to analyze a few results
   max-alarms: 16
 
+  # this can be used to limit the size of the callstack when searching for contexts under which a function can be
+  # called.
+  max-context: 20
```

Comment:
Maybe naming it `max-calling-context-length` makes a bit more sense to distinguish it from other kinds of contexts?

Reply:
I could call it `max-entrypoint-context-size` which specifies a little more where it applies? There's only one type of context so far, and for the full description you'd need to read the doc anyway (unless we give it a long name like `max-calling-context-length-for-entrypoint-disambiguation`)